### ISSUE TITLE
Certificat error with exocli on Windows

### ISSUE THREAD

**Issue by martialt** [exoscale/cli#189]
Hi,

I get this error when I try to list my buckets from exocli.exe on Windows server.
```
C:\Users\Administrator\Downloads\exocli>exo.exe sos list
error: Get https://sos-ch-dk-2.exo.io/: x509: certificate signed by unknown authority
```
Regards,
Martial

**Comment by falzm** [comment]
Hi, thank you for reporting this issue. Can you please provide details on your environment (Windows version, `exo` version...)?

**Comment by martialt** [comment]
Sure.
I used this Exoscale template "Windows Server 2019" and exocli 1.5.1.

**Comment by falzm** [comment]
Hi @martialt, just to let you know that we've successfully reproduced the issue on our side and are working on a fix right now, stay tuned.

**Comment by falzm** [comment]
Hi @martialt, we have identified and fixed the issue, we've tested it on Windows and it seems to resolve the problem.

Here is a test binary of the patched `exo` command, could you please run it on your end and confirm that it works as expected? You can find the bugfix details in PR #192, however long story short: you'll need a file containing the certificates chain for the Exoscale SOS API and provide it to your `exo sos ...` commands; I've also attached this file to the issue for convenience.

* [exo-8a24af6.zip](https://github.com/exoscale/cli/files/3777561/exo-8a24af6.zip)
* [exoscale-sos_certificates.zip](https://github.com/exoscale/cli/files/3777577/exoscale-sos_certificates.zip)

Let us know how it goes :)

**Comment by martialt** [comment]
Hi @falzm, works well with the new option.
```
C:\Users\Administrator\Downloads\exo-8a24af6>exo-8a24af6.exe sos --certs-file ..\exoscale-sos_certificates\exoscale-sos.pem list
[2019-10-22 13:10:47 UTC] [ch-gva-2]    0 B  bar/
```

Thank you.

**Comment by falzm** [comment]
@martialt great, thank you for the feedback. This fix will be included in a new CLI release soon.

**Comment by falzm** [comment]
@martialt the fix has been shipped in the latest 1.6.0 version release: https://github.com/exoscale/cli/releases/tag/v1.6.0

**Comment by martialt** [comment]
@falzm 
OK. Thank you.
